src/src/rx-serial/SerialCRSF.cpp

```diff
@@ -36,51 +35,53 @@ void SerialCRSF::sendLinkStatisticsToFC()
             _fifo.push(crc);
         }
     }
-#endif // DEBUG_CRSF_NO_OUTPUT
 }
 
 uint32_t SerialCRSF::sendRCFrameToFC(bool frameAvailable, uint32_t *channelData)
 {
-    if (!frameAvailable)
-        return DURATION_IMMEDIATELY;
+    if (!OPT_CRSF_RCVR_NO_SERIAL)
+    {
```

Comment:
We don't need these checks because in the setup section where we choose the serial provider, if `OPT_CRSF_RCVR_NO_SERIAL` is set then it chooses `SerialNOOP`.